_posts/2024/2024-01-15-the-first-five-questions.md

```diff
@@ -0,0 +1,110 @@
+For example,
+rather than presenting some rules for running meetings,
+a lesson could start by asking,
+"Why do have so many meetings and why do so many of them suck?"
+and use that to motivate discussion of things like
+why large social structures tend to become hierarchical
+and how who's allowed to interrupt whom is tied to perceived social status.
+
+So:
+suppose we have six useful hours in a day of teaching and learning
+(i.e., eight hours minus setup, coffee breaks, lunch, and wrap-up).
+At 45 minutes per episode,
+that gives us eight episodes per day
+or sixteen episodes in total.
+What should the motivating questions for each episode be?
+I've listed four below,
+and I'd like to set up an hour-long call later this month
+for anyone who's interested in expanding on them,
+replacing them with better ones,
+or suggesting another approach entirely.
+If you'd like to take part,
+please [email me](mailto:{{site.author.email}}) and I'll send you an invitation.
+
+Why do we have so many meetings?
+:   Coordination costs rise as the square of the number of people,
+    so large groups must (a)
+    make work uniform
+    so that sets of people are effectively interchangeable (e.g., assembly lines and typing pools),
+    (b) create sub-groups and invest effort in coordinating their work,
+    or (c) allow sub-groups to work independently and pay the cost of duplicated or contradictory effort.
+
+Why do so many meetings suck?
+:   Most people don't seek to maximize profit, but to maximize social standing within their peer group.
+    Some people do this by being helpful, but others try to do it by establishing social dominance.
+    Their meetings turn into one-up displays,
+    which often take the form of interrupting people they perceive as having lower social status.
+    Social norms (such as meeting moderation rules) exist in part to constrain such behavior.
+
+How should software developers be assessed?
+:   Exams and performance reviews are facts of life in academia and industry.
+    They exist for good reasons—some people truly are more skilled or more productive than others—but
+    every quantitative metric for comparing people will inevitably be gamed,
+    while every qualitative metric is subject to various biases.
+    One response is to allow people to define their own victory conditions,
+    but that makes it difficult or impossible to compare people.
+
+Who owns the software I write?
+:   Depending on your circumstances,
+    the software you write may belong to your employer
+    even if you wrote it on your own time using your own computer
+    (because they provided the training you needed to write it),
+    or they may only have an interest in it if you used their physical resources,
+    or it may be entirely yours as long as you wrote it on your own time.
+    The answer depends on your venue's rules about intellectual property;
```

Comment:
What you say venue’s rules what do you mean? 

At least in Argentina there is a intellectual property law that rules who owns the software. For example, if you program you are always the author but who have the rights is different if you are and employee or not.

Reply:
The question "Who owns the software I write?" is one that, in my mind, leads to questions about ethics or law more than social sciences. This question perhaps offers a way to get buy-in from CS faculty more than the others.

Reply:
I was thinking about aiming this at working programmers rather than at CS faculty; I've added a note about Frank (prototypical audience member) that I hope clarifies that.